owl/SOMA-OBJ.owl

```diff
@@ -971,6 +971,28 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#ArtificialAgent -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#ArtificialAgent">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalAgent"/>
+        <rdfs:comment>A physical object with artificial characteristics, which can perform actions to achieve desired goals, typically has sensors and/or actuators.
+
+There can be non-physical artificial agents such as software programs but they are not considered here in the scope of artificial agent.</rdfs:comment>
+        <rdfs:label>Artificial agent</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#FunctionalPart -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#FunctionalPart">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalBody"/>
+        <rdfs:comment>Parts of an agent or an artifact are considered as functional parts.</rdfs:comment>
```

Comment:
there should be an axioms that asserts this, i.e. that *FunctionalPart isPartOf only Agent or Artifact*